CHANGELOG.md

```diff
@@ -3,8 +3,10 @@
 All notable changes to this project will be documented in this file. This library adheres to a versioning policy described in [the README](./README.md#versioning). The public API of this library consists of the functions exported in [h3core.js](./lib/h3core.js).
 
 ## [Unreleased]
+### Added
+- Added `h3GetFaces` binding (#54)
 ### Changed
-- Updated the core library to v3.4.4 (#52)
+- Updated the core library to v3.5.0 (#54)
```

Comment:
I'd suggest preserving both PR numbers in the CHANGELOG if not both lines.

Reply:
Sure, will update.